span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -159,7 +167,7 @@ public RawSpan convert(Span jaegerSpan) throws Exception {
 
       // build raw span
       RawSpan rawSpan = rawSpanBuilder.build();
-      LOG.debug("Converted Jaeger span: {} to rawSpan: {} ", jaegerSpan, rawSpan);
```

Comment:
@tim-mwangi To be clear, what is the issue you're trying to address?

Reply:
Currently you cannot deserialize the logged rawspan into a RawSpan object for debugging purposes.